### PR TITLE
Fix/event announce

### PR DESCRIPTION
**Pull request by AverageMarcus**

### CODE REVIEW

**Comment by danielthepope** [comment]
Even asking "when is the next devops thames valley meetup?" queries DevOps Oxford. More thinking required I'm afraid.

**Comment by AverageMarcus** [comment]
Feel free to take stab at it. Otherwise I'll see if I get time in the morning. 

Possibly thinking to ignore all those that are out of oxford unless the name matches.

File: scripts/new-meetup.js [hunk 1]
```diff
@@ -16,7 +16,8 @@ module.exports = function(robot) {
   var groups = require('../meetup-groups.json');
   var allGroupIds = Object.keys(groups).join('%2C');
 
-  var meetupURL = `https://api.meetup.com/2/events?offset=0&format=json&limited_events=False&group_id=${allGroupIds}&only=created%2Ctime%2Cevent_url%2Cname%2Cdescription%2Cyes_rsvp_count%2Crsvp_limit%2Cgroup&photo-host=secure&page=20&fields=&order=time&status=upcoming&desc=false&key=${API_KEY}`;
+  var meetupURL = `https://api.meetup.com/2/events?offset=0&format=json&limited_events=False&group_id=${allGroupIds}&only=created%2Ctime%2Cevent_url%2Cname%2Cdescription%2Cyes_rsvp_count%2Crsvp_limit%2Cgroup&photo-host=secure&page=200&fields=&order=time&status=upcoming&desc=false&key=${API_KEY}`;
+
```
Comment (omgmog) [inline scripts/new-meetup.js:20]: this change is just white space, right?

Comment (AverageMarcus) [reply scripts/new-meetup.js:20]: The newline? Yes. I did have a console log in there when debugging.

**Review by omgmog — CHANGES_REQUESTED** [review]
Looks like it'd work. 

Might want to be more explicit with that return statement so that it makes sense if/when we come back to it in the future? (e.g. `matchingGroups.length > 0` rather than implicit).

**Comment by AverageMarcus** [comment]
@omgmog Done

**Review by omgmog — APPROVED** [review]

**Pull request merged**